pymc/tests/distributions/test_distribution.py

```diff
@@ -416,3 +416,11 @@ def test_tag_future_warning_dist():
         with pytest.warns(FutureWarning, match="Use model.rvs_to_values"):
             value_var = new_x.tag.value_var
         assert value_var == "1"
+
+
+def test_autonaming():
+    """Test that random variable ends up with same name as assignment variable"""
+    with pm.Model():
+        x = pm.Normal(0.0, 1.0)
+
+    assert x.name == "x"
```

Comment:
Just so I can sleep well at night?

```suggestion
    with pm.Model() as m:
        x = pm.Normal(0.0, 1.0)

    assert x.name == "x"
    m["x"] = x
```

Reply:
Will do!

Reply:
Yup you read me correctly